health-services/project-factory/src/server/config/constants.ts

```diff
@@ -146,6 +147,21 @@ export const processTrackStatuses = {
     failed: "failed",
 }
 
+export const resourceDistributionStrategyTypes = [
+    "FIXED", "MIXED", "HOUSE_TO_HOUSE"
+]
+
+export const rolesForMicroplan = ["Microplan approver", "National microplan approver", "Population data approver", "National population data approver", "Facility catchment assigner", "National facility catchment assigner", "Microplan viewer"];
```

Comment:
These will be converted to codes by capitalizing and adding underscores. I’m writing them here in normal words because the user roles generation in the admin console requires them in this format. This way, it follows the flow for enriching user roles, and I won't need to write additional functions.